frontend/src/components/VideoPreview.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+function VideoPreview({ video }) {
+  return (
+    <figure className="video-preview">
+      <img src={video.thumbnail} alt={video.title} />
+      <div className="video-info">
+        <h3>{video.title}</h3>
+        <figcaption>{video.description}</figcaption>
```

Comment:
figcaption should be a direct child of figure : your div shoudl be a figcaption. the description should be a paragraph

![image](https://github.com/WildCodeSchool/2023-02-JS-Reims-project-3-origins-digital/assets/10051208/77b39e34-a97e-4e7e-b966-353ec339f20b)